services/src/test/scala/cromwell/pipeline/service/ServiceTestMode.scala

```diff
@@ -0,0 +1,7 @@
+package cromwell.pipeline.service
+
+trait ServiceTestMode
+
+case object Success extends ServiceTestMode
+case object WithException extends ServiceTestMode
+case object WithReturnNone extends ServiceTestMode
```

Comment:
Why do we need it? Shouldn't `Success` be enough?
If you think that you really need then refactor hierarchy a little bit. I propose to make
```scala
sealed trait TestMode
sealed trait Success extends TestMode
sealed trait Failure extends TestMode

final case class WithException(exc: Throwable) extends Failure
final case class Calculated extends Success // previous Success
final case class ConstNone extends Success
```

Reply:
I added this because "get configuration by project id" testcase in our `ProjectConfigurationControllerTest` treats `None` case separately. The same goes with `signIn` method in` AuthService`. How can I handle it better?
Refactored `TestMode` according to your remarks, thanks!